camera/camera_system.py

```diff
@@ -0,0 +1,41 @@
+"""
+カメラシステムモジュール.
+
+@author: Takahiro55555
+"""
+
+
+class CameraSystem:
+    """カメラシステムクラス."""
+
```

Comment:
pythonのドキュメントコメントについて調べてみたけど，ファイルそのものに対するドキュメントってどう残すのが一般的なのかちょっとわからなかった
Doxygenみたいにしてしまってもいいんではないかと思うんだけどどうでしょう

クラスの説明は欲しいけどそれは9行目とかに挿入するべきなのかな？

```suggestion
"""
@file camera_system.py
@brief カメラシステムモジュール
@author: Takahiro55555
"""


class CameraSystem:
    """カメラシステムクラス
    カメラシステム全体を制御する
    """
```
クラス説明間違ってたらゴメン

Reply:
Docstringに関しては以下の記事を参考に、GoogleスタイルのDocstringに統一することにしました。
https://qiita.com/11ohina017/items/118b3b42b612e527dc1d

また、作成者名は記述したほうが良いと考え、著者名の部分のみDoxygen形式で記述するようにしてみました。